Review & implement suggestions to Angular sign-in guide (#1952)
okta/okta-developer-docs: packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md

## Add an OpenID Connect Client

> * Sign in to the Okta Developer Dashboard, and select **Create New App**. <br/>

Reviewer: ```suggestion
* Sign in to the Developer Console, and select **Create New App**. <br/>
```

> You can sign in to the Admin Console using [https://login.okta.com](https://login.okta.com), and then click **Admin**.

Reviewer 2: ```suggestion
You can sign in to the Admin Console using <https://login.okta.com>, and then click **Admin**.
```

> You can sign in to the Okta Developer Dashboard through [https://login.okta.com](https://login.okta.com) and clicking `Admin` button on the Okta Dashboard.

Reviewer: ```suggestion
You can sign in to the Developer Console using [https://login.okta.com](https://login.okta.com), and then click **Admin**.
```

> You can sign in to Okta Developer Dashboard through [https://login.okta.com](https://login.okta.com) and clicking `Admin` button on Okta Dashboard or by signing in directly into Okta Developer Dashboard through URL which looks like [hxxps://dev-1235813-admin.okta.com](hxxps://dev-1235813-admin.okta.com).

Reviewer 2: @albertchen-okta confirm please, but I don't think we want to have any instructions for the Developer Console?

| Login redirect URIs  | `http://localhost:4200/login/callback`              |
| Logout redirect URIs | `http://localhost:4200/login`                       |
| Allowed grant types  | Authorization Code                                  |

> `${clientId}` placeholders further in this tutorial should be replaced by `Client ID` of created application.

Reviewer: ```suggestion
`${clientId}` placeholders further in this tutorial should be replaced by the `Client ID` of the created application.
```
